src/WebJobs.Extensions/Extensions/Timers/Scheduling/CronSchedule.cs

```diff
@@ -14,14 +16,41 @@ namespace Microsoft.Azure.WebJobs.Extensions.Timers
     public class CronSchedule : TimerSchedule
     {
         private readonly CrontabSchedule _cronSchedule;
+        private readonly ILogger _logger;
+
+        internal CronSchedule(CrontabSchedule schedule, ILogger logger)
+        {
+            _cronSchedule = schedule;
+            var cron = schedule.ToString();
+            var parts = cron.Split(' ');
```

Comment:
I think you may want to split like the HasSeconds ensuring we remove empty entries. E.g. maybe add a test case like "     
          * * * * * *" to your tests and see what happens?

Reply:
Good callout. I tested this just now and turns out we don't explicitly need to do that b/c we're looking at this after `CrontabSchedule` has already done all of its internal normalization. But I'm adding it anyway.